mobile/src/main/java/org/openhab/habdroid/core/connection/Connections.java

```diff
@@ -0,0 +1,43 @@
+package org.openhab.habdroid.core.connection;
+
+public final class Connections {
+    /**
+     * Returns any openHAB connection that is most likely to work on the current network. The
+     * connections available will be tried in the following order:
+     *  - LOCAL
+     *  - REMOTE
+     *  - CLOUD
+     *  - DISCOVERY
+     *
+     * The connection type ANY guarantees to return a Connection object, however, it may throw an
+     * exception of type ConnectionException
+     */
+    public static final int ANY = 0;
+    /**
+     * Returns a connection to a locally hosted openHAB server, which is most likely the instance
+     * configured in the settings. May or may not work on the network the device is currently
+     * connected to.
+     *
+     * This method can return null, if no Connection for a local openHAB server could be
+     * constructed. In this case, no exception with more information will be thrown.
+     */
+    public static final int LOCAL = 1;
```

Comment:
Maybe name it `TYPE_LOCAL`?

Reply:
Hmm, would that be a big benefit? I somehow don't care, so if you think, that one is better, I'll change it, I just thought, using Connections.LOCAL is already as descriptive as it needs to be :)

Reply:
Since all constants are type, we can leave it as it is now. But when we add some other category, we should add TYPE_ to order the varibales logically